libvast/vast/bloom_filter.hpp

```diff
@@ -110,11 +111,14 @@ class bloom_filter : detail::equality_comparable<
 
   template <class Inspector>
   friend auto inspect(Inspector& f, bloom_filter& x) {
-    auto result = f(caf::meta::type_name("bloom_filter"), x.hasher_, x.bits_);
-    // When deserializing into a vector that was already bigger than required,
-    // CAF will reuse the storage but not release the excess afterwards.
-    x.bits_.shrink_to_fit();
-    return result;
+    return f(caf::meta::type_name("bloom_filter"), x.hasher_, x.bits_,
+             caf::meta::load_callback([&]() -> caf::error {
+               // When deserializing into a vector that was already bigger than
+               // required, CAF will reuse the storage but not release the
+               // excess afterwards.
+               x.bits_.shrink_to_fit();
+               return caf::none;
+             }));
```

Comment:
Mind splitting this into a more readable form of code? This one-liner is quite bulky to process cognitively.

e.g.,

```cpp
auto load_callback = ...;
return f(..., load_callback);
```